dpypelines/pipeline/configuration.py

```diff
@@ -1,16 +1,17 @@
 import re
 
 from dpypelines.pipeline.dataset_ingress_v1 import dataset_ingress_v1
-from dpypelines.pipeline.shared.transforms.sdmx.v1 import (
-    sdmx_compact_2_0_prototype_1,
-    sdmx_sanity_check_v1,
-)
+from dpypelines.pipeline.generic_file_ingress_v1 import generic_file_ingress_v1
+from dpypelines.pipeline.shared.transforms.sanity_check import sdmx_sanity_check_v1
+from dpypelines.pipeline.shared.transforms.sdmx.v20 import sdmx_compact_2_0_prototype_1
+from dpypelines.pipeline.shared.transforms.sdmx.v21 import sdmx_generic_2_1_prototype_1
 
 # Set a regex pattern matching the `source_id` as `CONFIGURATION` dictionary key
 # All fields are required in order for a pipeline transform to run successfully
 CONFIGURATION = {
-    # This is an example of how to set configuration details for a specific source_id (`cpih` in this case)
-    "^cpih$": {
+    # This is an example of how to set configuration details for a generic source_id - ending with v2_0, v2_1 for sdmx 2.0 and 2.1 respectively, and
+    # generic source_id - ending with move for an sdmx file to be moved
+    "^.*_v2_0$": {
```

Comment:
you probably need to have an indicator that its sdmx and compact in here.

- what if its not compact, would it not need a different function?
- what if its v2_0 of something else? maybe we start using a different data format and end up with a source_id of new_cool_format_v2_0 - that'd end up getting processed as sdmx right?